manifests/n/Netbird/Netbird/0.37.2/Netbird.Netbird.locale.en-US.yaml

```
@@ -0,0 +1,57 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Netbird.Netbird
+PackageVersion: 0.37.2
+PackageLocale: en-US
+Publisher: Wiretrustee UG (haftungsbeschreankt)

```

```suggestion
Publisher: NetBird GmbH
```

This is the ARP publisher value from the MSI installation